jcl/src/openj9.dtfj/share/classes/com/ibm/jvm/j9/dump/extract/Main.java

```diff
@@ -45,11 +43,11 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
-import com.ibm.dtfj.addressspace.IAbstractAddressSpace;
 import com.ibm.dtfj.corereaders.Builder;
 import com.ibm.dtfj.corereaders.ClosingFileReader;
 import com.ibm.dtfj.corereaders.DumpFactory;
 import com.ibm.dtfj.corereaders.ICoreFileReader;
+import com.ibm.dtfj.addressspace.IAbstractAddressSpace;
```

Comment:
Please keep the import list sorted (i.e. move this before line 49).

Reply:
Perhaps a line number wasn't the best way to explain where it belongs. It should appear before `Builder`:
```java
import com.ibm.dtfj.addressspace.IAbstractAddressSpace;
import com.ibm.dtfj.corereaders.Builder;
```